docs/pages/deployment/pex.rst

```diff
@@ -0,0 +1,92 @@
+.. _pex:
+
+Presentation definition mapping
+###############################
+
+Wallet functionality uses `Presentation Definitions <https://identity.foundation/presentation-exchange/>`_ to determine the required credentials for a given presentation request.
+An OAuth2 authorization request uses scopes to determine the required permissions for a given request.
+The mapping between scopes and presentation definitions is defined in a configuration file or by a policy backend.
+
+Configuration
+*************
+
+The Nuts config supports two ways to define the mapping between scopes and presentation definitions:
+- using a file-based configuration
+- using a policy backend
+
+The file-based configuration is a simple way to define the mapping between scopes and presentation definitions.
+It can be used for simple use cases where the mapping is static and does not change often.
+
+To use file-based configuration, you need to define the path to a directory that contains policy configuration files:
```

Comment:
this section is the most relevant part for 95% percent of the parties: individual vendors aren't going to write their own PEX-scope-mapping, as it's dictated by the use case.